addons/resource/static/src/components/avatar_card_resource/avatar_card_resource_popover.js

```diff
@@ -0,0 +1,61 @@
+/** @odoo-module **/
+
+import { onWillStart } from "@odoo/owl";
+import { useService } from "@web/core/utils/hooks";
+import { useOpenChat } from "@mail/core/web/open_chat_hook";
+import { AvatarCardPopover } from "@mail/discuss/web/avatar_card/avatar_card_popover";
+
+
```

Comment:
Indeed, I think 2 lines are needed between imports and class declaration. Please check that is is the case for all js files you created :)